datafusion/expr-common/src/type_coercion/binary.rs

```diff
@@ -1550,6 +1552,62 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_get_wider_type_with_utf8view() {
+        assert_eq!(
```

Comment:
Not this kind of test, but a valid sql query that goes through the coercion function

Reply:
@jayzhan211 Correct me if I'm wrong, but this seems to be a problem with arrow casting from numerics to utf8view, the same can be said for this, #13366. However correct me if I'm wrong, I can't seem to find a sql query that triggers it.

Reply:
I think get_wider_type is actually only used by ArrayConcat

![image](https://github.com/user-attachments/assets/1c86bf92-df63-4268-a187-4dd982fc1c4f)

Reply:
I mean `binary_to_string_coercion `